### PR TITLE
Release new versions **NON-BREAKING**

### PR DESCRIPTION
**Pull request by Xaeroxe**
@torkleyy Fixed many warnings for our crates in https://github.com/amethyst/amethyst/pull/324 so I propose we send new versions to cargo.

### CODE REVIEW

File: amethyst_assets/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "amethyst_assets"
-version = "0.1.0"
+version = "0.1.1"
```
Comment (torkleyy) [inline amethyst_assets/Cargo.toml]: Let's also use this version when specifying `amethyst_assets` as dependency in our core crate.

**Comment by torkleyy** [comment]
I don't think this is worth a release, but I'm sure we can release a new version soon, so let's leave it open.

**Comment by Xaeroxe** [comment]
Since we're delaying merging this I just want to make it clear to any future readers this only increments the patch versions, not the minor versions.

**Comment by Xaeroxe** [comment]
There haven't really been any interesting non breaking changes made since 0.5.1 was released, and all work from this point forward will be breaking it seems, as I doubt anyone is interested in rebasing onto 0.5.  so we can either release 0.5.2 now or just never put one out at all.

**Comment by torkleyy** [comment]
I don't think we need a 0.5.2 unless something is horribly broken there.

**Pull request closed without merging**